content/docs/handling-events.md

```diff
@@ -1,72 +1,72 @@
 ---
 id: handling-events
-title: Handling Events
+title: Obsługa zdarzeń
 permalink: docs/handling-events.html
 prev: state-and-lifecycle.html
 next: conditional-rendering.html
 redirect_from:
   - "docs/events-ko-KR.html"
 ---
 
-Handling events with React elements is very similar to handling events on DOM elements. There are some syntactic differences:
+Obsługa zdarzeń w Reakcie przypomina tę z drzewa DOM. Istnieje jednak kilka różnic w składni:
```

Comment:
`Obsługa zdarzeń w Reakcie przypomina tę z drzewa DOM` - moja sugestia to zamienić to na `Obsługa zdarzeń w Reakcie jest bardzo podobna do obsługi zdarzeń elementów drzewa DOM`. Nie jestem pewien czy w Twoim tłumaczeniu poprawnie jest użyć "tę" czy "tą", dlatego też sugeruję inne tłumaczenie.

Reply:
Na pewno "tę". Zastanawiam się, czy powtórzenie w Twoim tłumaczeniu nie będzie zbyt widoczne?

Reply:
Fakt w zasugerowanej przeze mnie zmianie występuje powtórzenie. Jednak tłumacząc z oryginału także występuje w tym miejscu powtórzenie. Po prostu jakby się tak zastanowić i przeczytać to zdanie to mi osobiście słowo `przypomina` nie sugeruje że coś jest podobne. W tym przypadku w angielskim tłumaczeniu zastosowane jest także słówko 'very' które w wersji z przypomina zostało usunięte a to podkreśla i daje do zrozumienia czytelnikowi że `obsługa zdarzeń jest bardzo podobna do ...` a nie tylko `przypomina`. 👍 Tak jak @jakubdrozdek napisał, prosimy o wypowiedź @cvqprs @TeoTN.

Reply:
To może lepiej: `Obsługa zdarzeń w Reakcie jest bardzo podobna do tej z drzewa DOM.`?

Reply:
Jestem za 👍